include/eve/arch/cpu/logical_wide.hpp

```diff
@@ -267,11 +256,10 @@ namespace eve
     //==============================================================================================
     friend std::ostream &operator<<(std::ostream &os, logical const &p)
     {
-      constexpr auto sz = sizeof(storage_type)/sizeof(logical<Type>);
-      auto that = bit_cast( p, as_<std::array<logical<Type>,sz>>());
-      os << '(' << that[0];
-      for(size_type i = 1; i != p.size(); ++i) os << ", " << that[i];
-      return os << ')';
+      auto that = p.bitmap();
+      os << '(' << std::boolalpha << that[0];
+      for(size_type i = 1; i < p.size(); ++i) os << ", " << that[i];
+      return os << std::noboolalpha << ')';
```

Comment:
I dunno how much you care, but this seems to change the global state of things (if you had boolapha true, it will get it to false).

I would just not bother with boolalpha maybe and then you will just support it:if you want true false - you get true false, otherwise - numbers.

Reply:
Chanegd to not depend on boolalpha state